src/components/LayerCollection/LayerPOLine.js

```diff
@@ -167,14 +167,19 @@ function LayerPOLine({
 
     return memoizedMutator.poLines
       .POST(newLine)
-      .then(() => openOrder(saveAndOpen))
-      .then(() => {
+      .then(({ id: poLineId }) => openOrder(saveAndOpen, poLineId))
+      .then(lineIdResponse => {
         sendCallout({
           message: <SafeHTMLMessage id="ui-orders.line.create.success" />,
           type: 'success',
         });
+
+        const pathname = lineIdResponse
```

Comment:
Could you please explain the reason or result for this `pathname` ? I can't get it

Reply:
The idea is that if user just create new POL, after saving he will see new POL details. But if he clicks save and open order button he will be redirected to PO details

Reply:
agree